cocos/editor-support/cocostudio/WidgetReader/NodeReader/NodeReader.cpp

```diff
@@ -36,10 +36,10 @@ using namespace flatbuffers;
 
 namespace cocostudio
 {
-    const char* Layout_PositionPercentXEnabled = "PositionPercentXEnable";
-    const char* Layout_PositionPercentYEnabled = "PositionPercentYEnable";
-    const char* Layout_PercentWidthEnable = "PercentWidthEnable";
-    const char* Layout_PercentHeightEnable = "PercentHeightEnable";
+    const char* Layout_PositionPercentXEnabled = "PositionPercentXEnabled";
```

Comment:
Why should add the “d”? 
Is wrong? ( Why just add some fields? )
How to ensure the forward compatibility?